crates/spirv-builder/Cargo.toml

```
@@ -22,21 +22,23 @@ default = ["use-compiled-tools"]
 # Compile `rustc_codegen_spirv`, allows constructing SpirvBuilder without
 # explicitly passing in a path to a compiled `rustc_codegen_spirv.so` (or dll)
 rustc_codegen_spirv = ["dep:rustc_codegen_spirv"]
+# Inclide target spec json files, allows constructing SpirvBuilder without
+# explicitly passing a path to the target spec json
+include_target_specs = ["dep:rustc_codegen_spirv-target-specs"]
```

I Ctrl+F'd `_target_specs` and found this one, sorry I missed it the first time.
```suggestion
include-target-specs = ["dep:rustc_codegen_spirv-target-specs"]
```